### PR TITLE
Add Benchmarks and function for evaluation

### PR DESCRIPTION
**Pull request by Reindulger**
## Description

- Add "Text+Image ➡️ Text" Benchmarks.
- Add function for models_pk of evaluation.
- Fix some bugs for evaluation and configs.

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [ ] I have read the [[CONTRIBUTION](https://github.com/PKU-Alignment/align-anything/blob/HEAD/.github/CONTRIBUTING.md)](https://github.com/PKU-Alignment/align-anything/blob/HEAD/.github/CONTRIBUTING.md) guide. (**required**)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly. (*required for a bug fix or a new feature*)
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by XuyaoWang — APPROVED** [review]
LGTM.

**Review by htlou — APPROVED** [review]
LGTM, but I don't have time to look through all the details.

File: README_zh-CN.md [hunk 1]
```diff
@@ -297,19 +298,73 @@ class PKUSafeRLHF(Template):
...
+        --benchmark ${BENCHMARK} \
+        --output_dir ${OUTPUT_DIR} \
+        --generation_backend ${GENERATION_BACKEND} \
+        --model_id ${MODEL_ID} \
+        --model_name_or_path ${MODEL_NAME_OR_PATH} \
+        --chat_template ${CHAT_TEMPLATE}
+done
+~~~
+
+- `BENCHMARK`: 用于评估模型性能的一个或多个评估基准或数据集。例如，`("POPE" "MMBench")` 可用于在 POPE 和 MMBench 数据集上评估模型。列表中的每个基准将按顺序处理。
```
Comment (Gaiejj) [inline README_zh-CN.md:319]: BENCHMARKS?

File: align_anything/configs/evaluation/benchmarks/pope.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    # Pretrained model name or path
+    model_name_or_path: null
+    # Chat template
+    chat_template: null
+    # Whether to trust remote code
+    trust_remote_code: True
+    # The max token length
+    max_length: 1024
+    # The max new tokens for generation
+    max_new_tokens: 512
```
Comment (Gaiejj) [inline align_anything/configs/evaluation/benchmarks/pope.yaml:50]: why we set them together?

**Review by cby-pku — CHANGES_REQUESTED** [review]
Wait for test.

File: README_zh-CN.md (hunk 1 above)
Comment (Reindulger) [reply README_zh-CN.md:319]: Yeah, so careless was I.

File: align_anything/configs/evaluation/benchmarks/pope.yaml (hunk 2 above)
Comment (Reindulger) [reply align_anything/configs/evaluation/benchmarks/pope.yaml:50]: Yeah, it's redundant. We will fix it.

**Pull request merged**